docs/spec_draft.md

```diff
@@ -438,16 +438,16 @@ tensor. More formally,
 
 ### Inputs
 
-| Name        | Type                                              |
-|-------------|---------------------------------------------------|
-| `inputs`    | variadic number of tensors of any supported types |
-| `dimension` | `si64`                                            |
+| Name        | Type                                                      |
+|-------------|-----------------------------------------------------------|
+| `inputs`    | variadic number of tensors of any supported element types |
```

Comment:
Should we perhaps go the other way around and contract "element types" just "types"?

Reply:
We also have Tensor as a supported type, hence the "element type"

Reply:
The "Types" section says "Tensor types... Element type E is any one of the supported element types mentioned above", so "any supported type" effectively means "any supported element type".